pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -560,6 +562,60 @@ func TestMakePodSpec(t *testing.T) {
 					},
 				},
 			})),
+	}, {
+		name: "explicit true service links",
```

Comment:
I'd drop these tests - theres nothing unique being exercised in `deploy.go`

Reply:
well it verifies we're not using a constant, but rather it indeed follows the CM setting.

Reply:
👍  - i realize the other PR didn't have a test for the change